app/controllers/application_controller.rb

```diff
@@ -107,4 +107,16 @@ def not_authorized
     flash[:notice] = "Sorry, you are not authorized to perform this action."
     redirect_to(root_url)
   end
+
+  def admin_email_change(user) # for casa admins
+    if user.saved_changes.include?("unconfirmed_email")
+      redirect_to edit_polymorphic_path(user), notice: "Confirmation Email Sent To #{user.role}."
```

Comment:
🏅  love this, great way to keep this clean

Reply:
Yay! Thank you! Big thanks to Code Climate for prodding me to go for it